### PR TITLE
Test linter

### PR DESCRIPTION
**Pull request by FevenSeyfu**
completed initial setup on branch test linter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,282 @@
...
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-world] <a name="practice tools like gitub flow and linters and best practices"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[hello-world]** is a...
```
Comment (anagudelogu) [inline README.md:64]: - [ ] Kindly customize your README with the actual information of your project. Remember that having professional code documentation will make your projects more attractive to people that want to collaborate on them and also will provide better guidance on how to interact with them. Please consider modifying all the relevant sections of your README (e.g. Description, Getting started, etc)! 🤓 🛠️

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @FevenSeyfu, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 

## Required Changes ♻️

- [ ] Please consider adding a descriptive pull request summary and a title describing the changes you implemented, it is part of professional code documentation. 🤓 🛠️ 

image

- [ ] I noticed that you didn't install linters locally in your project, please consider following this [entire step-by-step tutorial](https://github.com/microverseinc/linters-config/tree/master/html-css) on how to add linters to your HTML and CSS projects (You're missing `.hintrc` and `.stylelintrc.json` files). 🤓 🛠️ 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (FevenSeyfu) [reply README.md:64]: pull request for practicing GitHub best practices specifically the pull request process and commenting best practice, added a more specific comment, and added linters to the local environment.

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @FevenSeyfu 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✅ Great use of GitHub flow. 💯
✅ Your project is documented professionally. 📜

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

 - [OPTIONAL] Your linters checks are running, but one of them is failing. It would be nice to fix all linters checks before submitting a PR for approval. For the next projects, please don't forget to check those linters and fix all of them.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**